Sources/SwiftProtobuf/JSONEncodingVisitor.swift

```diff
@@ -226,9 +233,12 @@ internal struct JSONEncodingVisitor: Visitor {
   }
 
   mutating func visitRepeatedUInt64Field(value: [UInt64], fieldNumber: Int) throws {
+    let options = self.options
     try _visitRepeated(value: value, fieldNumber: fieldNumber) {
       (encoder: inout JSONEncoder, v: UInt64) in
-      encoder.putUInt64(value: v)
+      options.alwaysPrintInt64sAsNumbers
```

Comment:
Might make sense to move the condition outside the loop.

Reply:
Was following the pattern used by `alwaysPrintEnumAsInts` but it's indeed a bit different as it also has a condition on the enum having a value for property `name`.
Will update that.
Thank you.

Reply:
It does come with some logic duplication though, to a performance benefit I personally consider low:
```swift
if options.alwaysPrintInt64sAsNumbers {
      try _visitRepeated(value: value, fieldNumber: fieldNumber) {
        (encoder: inout JSONEncoder, v: Int64) in
        encoder.putNonQuotedInt64(value: v)
      }
    } else {
      try _visitRepeated(value: value, fieldNumber: fieldNumber) {
        (encoder: inout JSONEncoder, v: Int64) in
        encoder.putInt64(value: v)
      }
    }
```

Or maybe you had a better idea syntax-wise?
Let me know what you think and I'll take care of it.

Reply:
Hi @thomasvl. Sorry for the ping but any news on this? I'd love for this to be merged and released so I could stop using my fork. Thank you 🙏.

Reply:
As a rule, we generally do split things like this into two loops as @thomasvl suggested.  Although the performance benefit may seem modest to most folks, we have some users who are _very_ passionate about getting every bit of performance improvement they can.

Reply:
I've made the requested changes.
Thanks for the insights on performance requirements 🙏 .